run/CosmologySimulation/dm_only/dm_only.enzo

```diff
@@ -13,6 +13,7 @@ SelfGravity                = 1       // gravity on
 TopGridGravityBoundary     = 0       // Periodic BC for gravity
 LeftFaceBoundaryCondition  = 3 3 3   // same for fluid
 RightFaceBoundaryCondition = 3 3 3
+NumberOfGhostZones         = 8
```

Comment:
Does this test need 8 ghost zones? I imagine that it'd add unnecessary computation without much more accuracy.

Reply:
removed